Graph Theory/DFS/DFS.py

```diff
@@ -46,12 +46,12 @@ def DFS(self, v):
 # Create a graph given  
 # in the above diagram 
 g = Graph() 
-g.addEdge(0, 1) 
-g.addEdge(0, 2) 
-g.addEdge(1, 2) 
-g.addEdge(2, 0) 
-g.addEdge(2, 3) 
-g.addEdge(3, 3) 
+E = int(input("Enter the number of edges:"))
+
+for i in range(0,E):
+  x, y = input("Enter a edges value: ").split()
+  g.addEdge(x, y) 
+
   
 print("Following is DFS from (starting from vertex 2)") 
 g.DFS(2) 
```

Comment:
Can the starting vertex be dynamic?

Reply:
I cannot find the new commit, so have mentioned the old commit here. (Thus we can see outdated)

Reply:
i made it dynamic.
please, review it

Reply:
4358ddaf4ac41a10cf13d35070e43f3daed6821c